### ISSUE TITLE
Circular inclusion

### ISSUE THREAD

**Issue by truebluepl** [GeorgRottensteiner/C64Studio#121]
Hello Georg,
Is there any possibility to include the same file twice?
I have file (some sort of table) which should be included several times, but compiling process generates "circular inclusion" error.

**Comment by truebluepl** [comment]
I'm sorry, I forgot about macros:) They solve problem.

**Comment by GeorgRottensteiner** [comment]
Additionally there's the new !src "file", once directive.

You can place that several times, and C64Studio will only actually include it once.
You have to use the ,once argument on every occasion for it to work though.

**Comment by truebluepl** [comment]
If I understood correctly, it's only "prevents" from multiple including - multiple included file is joined only once, right?

**Comment by GeorgRottensteiner** [comment]
Exactly. It just removes the need to futz around with making a manual include guard via defines.
